integration/util_test.go

```diff
@@ -483,6 +512,17 @@ networks:
 `, nw)
 	}
 
+	zotConfigFile, err := testutil.ApplyTextTemplate(zotConfigTemplate, zotConfigStruct{
```

Comment:
Just want to note, this is called every single time a new shell is created. It is inconsequential operationally — most shells will just ignore this file — but my test is the only one that needs this config, adding an option for this (in a separate commit, perhaps?) seems to make more sense logically.

Reply:
Yeah, personally I think we should move away from passing registry specific configs as environment variables in the [template](https://github.com/awslabs/soci-snapshotter/blob/main/integration/util_test.go#L169-#L176) and instead do something similar to what we did with Zot (having a separate config file and volume mounting). The Docker registry supports [reading](https://github.com/distribution/distribution/blob/main/docs/configuration.md#overriding-the-entire-configuration-file) a config from a file, so this should be possible. 

We can then volume mount the specific config file for the registry that gets passed in. I don't think it's really a blocker for this though.

Reply:
Makes sense, I suppose we can modify the header to add options depending on which registry config we needed instead of attaching the same variables for every image. I could look into refactoring that in the future.